lib/trestle/active_storage/controller_concern.rb

```diff
@@ -5,10 +5,33 @@ module ControllerConcern
 
       included do
         before_action :define_attachment_accessors, only: [:show, :edit, :update, :destroy]
+        before_action :filter_has_many_attachments, only: [:update]
         after_action :purge_attachments, only: [:update]
+        after_action :attach_attachments, only: [:update]
       end
 
       private
+      def filter_has_many_attachments
+        return if admin.active_storage_fields.empty?
+
+        @removed_attachments = Hash.new
+        admin.active_storage_fields.each do |field|
+          if instance.send(field).respond_to?(:each)
+            if params.key?(admin.parameter_name)
+              @removed_attachments[field] = params[admin.parameter_name].delete field.to_sym
+            end
+          end
+        end
+      end
+
+      def attach_attachments
+        return if @removed_attachments.nil? || @removed_attachments.empty?
```

Comment:
I think you could use `blank?` since rails is a dependency of trestle-active_storage, so this method should be available :)
```suggestion
        return if @removed_attachments.blank?
```